doc/pyplots/boxplot_demo.py

```diff
@@ -4,62 +4,68 @@
 # Example boxplot code
 #
 
-from pylab import *
+import numpy
+import matplotlib.pyplot as plt
+
+numpy.random.seed(0)
 
 # fake up some data
-spread= rand(50) * 100
-center = ones(25) * 50
-flier_high = rand(10) * 100 + 100
-flier_low = rand(10) * -100
-data =concatenate((spread, center, flier_high, flier_low), 0)
+spread = numpy.random.rand(50) * 100
+center = numpy.ones(25) * 50
+flier_high = numpy.random.rand(10) * 100 + 100
+flier_low = numpy.random.rand(10) * -100
+data = numpy.concatenate((spread, center, flier_high, flier_low), 0)
 
-# basic plot
```

Comment:
I added them to the figures as titles -- figure that just as informative when reading the code.